Scripts/AwsEC2GetPublicSGRules/AwsEC2GetPublicSGRules.py

```diff
@@ -0,0 +1,138 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import copy
+
+
+def get_dict_value(data, key):
+    """ Returns dict value for a given key (case insensitive) """
+    for key_name in data.keys():
+        if key_name.lower() == key.lower():
+            return data[key_name]
+
+    return None
+
+
+def raise_error(error):
```

Comment:
use `return_error` instead of this function.

Reply:
I updated the script to use return_error